packages/terra-button-group/package.json

```diff
@@ -27,11 +27,13 @@
   "peerDependencies": {
     "react": "^15.4.2",
     "react-dom": "^15.4.2",
-    "terra-mixins": "^1.0.0"
+    "terra-mixins": "^1.0.0",
+    "terra-button": "^0.x"
```

Comment:
It looks like this missed was missed during the terra-base PR: https://github.com/cerner/terra-core/pull/183

I'd sync up with @bjankord to see if this should have a peerDependency of terra-base, it can probably be added in a different pull request if it doesn't really relate to this one